js/src/wp-seo-replacevar-plugin.js

```diff
@@ -77,74 +334,150 @@
 	};
 
 	/**
-	 * Replaces %%parent_title%% with the selected value from selectbox (if available on page).
+	 * Replaces %%parent_title%% with the selected value from selectbox (if available on pages only).
 	 *
 	 * @param {String} data
 	 * @returns {String}
 	 */
 	YoastReplaceVarPlugin.prototype.parentReplace = function( data ) {
-		var parentId = document.getElementById( 'parent_id' );
+		var parentId = jQuery( '#parent_id, #parent' )[0];
 
-		if ( parentId !== null && typeof parentId.options !== 'undefined' && parentId.options[ parentId.selectedIndex ].text !== wpseoReplaceVarsL10n.no_parent_text ) {
-			data = data.replace( /%%parent_title%%/, parentId.options[ parentId.selectedIndex ].text );
+		if ( this.hasParentTitle() ) {
+			data = data.replace( /%%parent_title%%/, this.getParentTitleReplacement( parentId ) );
 		}
+
 		return data;
 	};
 
 	/**
-	 * Replaces separators in the string.
+	 * Checks whether or not there's a parent title available.
 	 *
-	 * @param {String} data
-	 * @returns {String}
+	 * @returns {boolean}
 	 */
-	YoastReplaceVarPlugin.prototype.replaceSeparators = function( data ) {
-		return data.replace( /%%sep%%(\s+%%sep%%)*/g, this.replaceVars.sep );
+	YoastReplaceVarPlugin.prototype.hasParentTitle = function() {
+		var parentId = jQuery( '#parent_id, #parent' )[0];
+
+		return ( isUndefined( parentId ) === false && isUndefined( parentId.options ) === false );
 	};
 
 	/**
-	 * replaces the excerpts strings with strings for the excerpts, if not empty.
+	 * Gets the replacement for the parent title.
 	 *
-	 * @param {String} data
-	 * @returns {String}
+	 * @param {int} parentId
+	 * @returns {*}
 	 */
-	YoastReplaceVarPlugin.prototype.excerptReplace = function( data ) {
-		if ( typeof this._app.rawData.excerpt !== 'undefined' ) {
-			data = data.replace( /%%excerpt_only%%/g, this._app.rawData.excerpt );
-			data = data.replace( /%%excerpt%%/g, this._app.rawData.excerpt );
+	YoastReplaceVarPlugin.prototype.getParentTitleReplacement = function( parentId ) {
+		var parentText = parentId.options[ parentId.selectedIndex ].text;
```

Comment:
This can be simplified with jQuery: `$( '#parent_id, #parent' ).val()` would do the trick.